pubsub/integration_test.go

```diff
@@ -735,7 +735,7 @@ func TestIntegration_UpdateSubscription_ExpirationPolicy(t *testing.T) {
 }
 
 // NOTE: This test should be skipped by open source contributors. It requires
-// whitelisting, a (gsuite) organization project, and specific permissions.
+// allowlisting, a (gsuite) organization project, and specific permissions.
```

Comment:
One word or two?

Reply:
I think one. On the style guide it is one word at least.